README.md

```diff
@@ -212,66 +224,63 @@ For example:
-<!DOCTYPE html>
-<html>
-<head>
-</head>
-<body>
-    <canvas id="canvas" width="300" height="300"></canvas>
-
-    <script src="lib/fabric.js"></script>
-    <script>
-        var canvas = new fabric.Canvas('canvas');
-
-        var rect = new fabric.Rect({
-            top : 100,
-            left : 100,
-            width : 60,
-            height : 70,
-            fill : 'red'
-        });
 
-        canvas.add(rect);
-    </script>
-</body>
-</html>
-```
+### Goals
 
-### Helping Fabric
+- Unit tested (1150+ tests at the moment, 79%+ coverage)
+- Modular (~60 small ["classes", modules, mixins](http://fabricjs.com/docs/))
+- Cross-browser
+- [Fast](https://github.com/kangax/fabric.js/wiki/Focus-on-speed)
+- Encapsulated in one object
+- No browser sniffing for critical functionality
+- Runs under ES5 strict mode
+- Runs on a server under [Node.js](http://nodejs.org/) (active stable releases and latest of current) (see [Node.js limitations](https://github.com/kangax/fabric.js/wiki/Fabric-limitations-in-node.js))
+- Follows [Semantic Versioning](http://semver.org/)
 
-- [Fabric on Bountysource](https://www.bountysource.com/trackers/23217-fabric-js)
-- [Fabric on CodeTriage](http://www.codetriage.com/kangax/fabric.js)
-- [Contributing](./CONTRIBUTING.md)
+### Supported browsers
 
-### Staying in touch
+- Firefox 4+
+- Safari 5+
+- Opera 9.64+
+- Chrome (all versions)
+- Edge (chromium based, all versions)
+- IE11 and Edge legacy, not supported. Fabric up to 5.0 is written with ES5 in mind, but no specific tests are run for those browsers.
 
-Follow [@fabric.js](http://twitter.com/fabricjs), [@kangax](http://twitter.com/kangax) or [@AndreaBogazzi](http://twitter.com/AndreaBogazzi) on twitter.
+You can [run automated unit tests](http://fabricjs.com/test/unit/) right in the browser.
 
-Questions, suggestions — [fabric.js on Google Groups](http://groups.google.com/group/fabricjs).
 
-See [Fabric questions on Stackoverflow](http://stackoverflow.com/questions/tagged/fabricjs),
-Fabric snippets on [jsfiddle](http://jsfiddle.net/user/fabricjs/fiddles/)
-or [codepen.io](http://codepen.io/tag/fabricjs).
+### More resources
 
-Fabric on [LibKnot](http://libknot.ohmztech.com/).
+- [Fabric.js on Twitter](https://twitter.com/fabricjs)
+- [Fabric.js on Google Groups](https://groups.google.com/group/fabricjs)
+- [Fabric.js on CodeTriage](https://www.codetriage.com/kangax/fabric.js)
+- [Fabric.js on Bountysource](https://www.bountysource.com/trackers/23217-fabric-js)
+- [Fabric.js on Stackoverflow](https://stackoverflow.com/questions/tagged/fabricjs)
+- [Fabric.js on jsfiddle](https://jsfiddle.net/user/fabricjs/fiddles/)
+- [Fabric.js on Codepen.io](https://codepen.io/tag/fabricjs)
+- [Presentation from BK.js](http://www.slideshare.net/kangax/fabricjs-building-acanvaslibrarybk)
+- [Presentation from Falsy Values](http://www.slideshare.net/kangax/fabric-falsy-values-8067834)
 
-Get help in Fabric's IRC channel — irc://irc.freenode.net/#fabric.js
 
 ### Credits
```

Comment:
This is old and outdated as well
I think it's irrelevant in the README - github does it why better.